### ISSUE TITLE
upper outline of the button in the modelview toolbar is not visible

### ISSUE THREAD

**Issue by alanrenmsft** [microsoft/azuredatastudio#9792]
in the screenshot below, the button 'Switch direction' is being focused, but the upper focus outline is not visible
![Screen Shot 2020-03-30 at 2 59 40 PM](https://user-images.githubusercontent.com/13777222/77966378-c04b2600-7297-11ea-8670-103ee00b7e7b.png)
